website_and_docs/content/blog/2023/java-8-support.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Java 8 support in Selenium"
+linkTitle: "Java 8 support in Selenium"
+date: 2023-06-09
+tags: ["selenium", "java"]
+categories: ["releases"]
+author: Diego Molina ([@diegofmolina](https://twitter.com/diegofmolina))
+description: >
+   On September 30, 2023, Java 11 will be the minimum version supported by Selenium.
+---
+
+
+As exciting as new technology is, sometimes it’s necessary to move on from our old favorites. 
+That’s why we’re announcing that **Selenium will stop supporting Java 8 on September 30, 2023**.
```

Comment:
Should we specify this is for both bindings and grid?

Reply:
I would say that grid can "move on it's own pace" and go further on.
(based on my docker based Selenium Grid Hub experience)
Grid can be pushed onwards, Bindings however can be a bit trickier to move faster up

Reply:
I will clarify that.

Reply:
I will resolve this as a new version was pushed. Please check again.